doc/stake-pool-operations/withdraw-rewards.md

```diff
@@ -26,13 +26,11 @@ You'll withdraw rewards into a payment.addr wich will pay for the transaction fe
     ----------------------------------------------------------------------------------------
     a82f8d2a85cde39118a894306ad7a85ba40af221406064a56bdd9b3c61153527     1         194054070
 
-
-
 ### Draft the withdraw transaction to transfer the rewards to a payment.addr
 
     cardano-cli shelley transaction build raw \
     --tx-in a82f8d2a85cde39118a894306ad7a85ba40af221406064a56bdd9b3c61153527#1
-    --tx-out $(cat payment.addr)+0
+    --tx-out $(cat payment.addr)+743882981
```

Comment:
This transaction is only used to estimate the transaction fees. The transaction output amount isn't relevant for that calculation. This can be changed back to 0.